ledger/ledger.go

```diff
@@ -1027,3 +1027,34 @@ func (l *Ledger) MaxTxSizePerBlock() int {
 func (l *Ledger) GetBaseDB() kvdb.Database {
 	return l.baseDB
 }
+
+func (l *Ledger) Truncate(ledgerLastID []byte, utxovmLastID []byte) error {
```

Comment:
Is ledgerLastID redundant?

Reply:
The purpose of retaining this parameter is  to deleting  incorrect blocks in the future  for next upgrade